packages/session/session.0.5.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/ocaml-session"
+bug-reports: "https://github.com/inhabitedtype/ocaml-session/issues"
+doc: "https://inhabitedtype.github.io/ocaml-session/"
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "1.0"}
+  "mirage-crypto"
+  "mirage-crypto-rng"
+  "base64" {>= "3.1.0"}
+  "ounit" {with-test & >= "1.0.2"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
```suggestion
```

The upstream project's Travis configuration doesn't run the tests for this package (and they don't link).